lib/pool.js

```diff
@@ -16,6 +16,14 @@ function parseUrl(url) {
   throw new Error('url is invalid: ' + url)
 }
 
+function unbuffer (key, value) {
```

Comment:
Not sure if worth it, but there is similar-ish type function in [log.js](https://github.com/mozilla/fxa-auth-server/blob/master/lib/log.js#L14). Maybe combined in common function?